### PR TITLE
chore: added parser function in find_face_in_video.py

### PR DESCRIPTION
**Pull request by GH1010**

### CODE REVIEW

File: data_preparation/find_face_in_video.py [hunk 1]
```diff
@@ -11,13 +11,34 @@
...
 import matplotlib.pyplot as plt
 import os
+import argparse
 
 savename = '1'
-videpath = './video/Arcane1.mp4'
+videopath = './video/Arcane1.mp4'
 outpath = './output/'
 predictorpath = './shape_predictor_68_face_landmarks.dat'
 black_width = 89
```
Comment (yutaochen1998) [inline data_preparation/find_face_in_video.py:20]: 上面这些可以删了

File: data_preparation/find_face_in_video.py [hunk 2]
```diff
@@ -144,11 +165,15 @@ def find_face(videoname, savename, detector, predictor):
...
 if __name__ == "__main__":
+
+    parser = TestOptions()
+    args = parser.parse()
+    print('*' * 98)
     
-    if not os.path.exists(outpath):
-        os.makedirs(outpath)
+    if not os.path.exists(args.outpath):
+        os.makedirs(args.outpath)
```
Comment (yutaochen1998) [inline data_preparation/find_face_in_video.py:174]: 可以直接：
```python
os.makedirs(args.outpath, exists_ok=True)
```

**Comment by SJXer** [comment]
Why does the model automatically correct the input face image when I test the code, so that a lot of face expression data will be lost.

**Comment by williamyang1991** [comment]
> Why does the model automatically correct the input face image when I test the code, so that a lot of face expression data will be lost.

Our trained pSp encoder on Z+ space cannot perfectly encode the content image. If the style transfer result more consistent with the content image is desired, one may use latent optimization to better fit the content image or using other StyleGAN encoders.

**Pull request closed without merging**